cmd/strapi-demo/createdemodata/start.go

```diff
@@ -58,11 +58,11 @@ func GetStartCmd() *cobra.Command {
 }
 func createStartCmd() *cobra.Command {
 	return &cobra.Command{
-		Use:   "create-demo-data",
-		Short: "Create demo data",
+		Use:   "start",
```

Comment:
why change to start

Reply:
all the other api which we run in local have start flag ... i forgot this had different flag so it took me some time to troubleshoot.  So to maintain consistency specially while troubleshooting, i feel we should stick to start

Reply:
but this not server

Reply:
it's different than other projects

Reply:
usually we use start for starting a server

Reply:
ok i will revert give me 5 min

Reply:
There is no server logic inside your start cmd command

Reply:
done